lib/src/10_auth/auth_screen.dart

```diff
@@ -128,9 +128,58 @@ class _FauiAuthScreenState extends State<FauiAuthScreen> {
     );
   }
 
+  Widget _buildTextBox(TextEditingController controllerName, String field,
+      String screen, Future<Widget> Function() submit) {
+    FauiPhrases phrase;
+    if (field == 'Email') {
+      phrase = FauiPhrases.EmailTextField;
+    } else if (field == 'Password') {
+      phrase = FauiPhrases.PasswordTextField;
+    }
+    return Focus(
+        child: TextField(
+          autofocus: (field == 'Email') ? true : false,
+          controller: controllerName,
+          decoration: InputDecoration(
+            labelText: resolvePhrase(phrase, field),
+          ),
+          onSubmitted: (s) {
+            submit();
+          },
+        ),
+        onFocusChange: (hasFocus) {
```

Comment:
you are doing the same thing in all these cases. Why you are checking condition at all? Can you just set _error to ""?